pepdbagent/db_utils.py

```diff
@@ -0,0 +1,173 @@
+import datetime
+import logging
+from typing import Any, Optional
+
+from sqlalchemy import (
+    BigInteger,
+    FetchedValue,
+    PrimaryKeyConstraint,
+    Result,
+    Select,
+    String,
+    event,
+    select,
+    TIMESTAMP,
+)
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.engine import URL, create_engine
+from sqlalchemy.exc import ProgrammingError
+from sqlalchemy.ext.compiler import compiles
+from sqlalchemy.orm import (
+    DeclarativeBase,
+    Mapped,
+    Session,
+    mapped_column,
+)
+
+from pepdbagent.const import POSTGRES_DIALECT
+from pepdbagent.exceptions import SchemaError
+
+_LOGGER = logging.getLogger("pepdbagent")
```

Comment:
I like to add `PKG_NAME` to `const.py` and use that to init the logger. Tiniest of suggestions.